api/db-cdr/generate-cdr/run-measurement-queries.sh

```diff
@@ -596,12 +596,12 @@ round((case when m1.value_as_number < p10_value then p10_value
      end),2)
      end) as string) as stratum_4,
 count(distinct p1.person_id) as count_value,
-0 as source_count_value
+count(distinct p1.person_id) as source_count_value
 from \`${BQ_PROJECT}.${BQ_DATASET}.measurement\` m1 join \`${BQ_PROJECT}.${BQ_DATASET}.person\` p1 on p1.person_id = m1.person_id
 join measurement_quartile_data on m1.measurement_concept_id=concept
 join \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c1 on m1.measurement_concept_id=c1.concept_id
 join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.unit_map\` um on m1.unit_source_value=um.unit_source_value
-where m1.measurement_concept_id != 0
+where m1.measurement_concept_id != 0 and measurement_source_concept_id = measurement_concept_id
```

Comment:
make this > 0 in case null with bad data

Reply:
yeah i would make it >= 0 in case there are real 0 values in there.

Reply:
So we want to count measurements that have concept id 0 ? What are you thinking we would do with that?

Reply:
i thought you were talking about the value as number.